easybuild/easyconfigs/d/DIRAC/DIRAC-22.0-foss-2021a.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'CMakeMake'
+
+name = 'DIRAC'
+version = '22.0'
+
+homepage = 'http://www.diracprogram.org'
+description = "DIRAC: Program for Atomic and Molecular Direct Iterative Relativistic All-electron Calculations"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True, 'i8': False}
+
+source_urls = ['https://zenodo.org/record/6010450/files/']
+sources = ['DIRAC-%(version)s-Source.tar.gz']
+checksums = ['c8c011b5bc2ca36a2ef76b2deb911638321655fcae1d1b146cacfa1b5f783711']
+
+builddependencies = [('CMake', '3.20.1', '', ('GCCcore', '10.3.0'))]
+
+dependencies = [
+    ('Python', '3.9.5', '', ('GCCcore', '10.3.0')),
+    ('HDF5', '1.10.7', '', ('gompi', '2021a'))
+]
+
+separate_build_dir = True
+
+configopts = ' -DCMAKE_Fortran_COMPILER=mpif90 '
```

Comment:
This should not be needed, with "usempi" true, FC = mpif90 (or mpiifort in the case of the intel toolchain)

Reply:
Hi @akesandgren. I just realized that when we set "usempi" to true and we use the intel toolchain, FC is set to be mpif90 (and we have mpicc and mpicxx for C and C++ compilers, respectively).

I mean, the mpiifort (and equivalents for c and cxx) are expected to be recognized automatically, but they are not. Is it probably due to a bug in the code?

Maybe the bug is in this file: https://gitlab.com/dirac/dirac/-/blob/master/cmake/extensions.py

All the cmake lines are here: https://gitlab.com/dirac/dirac/-/tree/master/cmake

Do you suggest to restore:

```
configopts  = ' -DCMAKE_Fortran_COMPILER=mpiifort '
configopts += '-DCMAKE_C_COMPILER=mpiicc '
configopts += '-DCMAKE_CXX_COMPILER=mpiicpc '
```

in the intel's *eb file? If it is something simple (I am not a CMake expert), perhaps it is better to solve the bug directly in the code.

CC: @boegel

Reply:
Yeah, try doing that for the intel version and check if it does the right thing all the way through.

Not quite sure what that extensions.py code is used for...